### PR TITLE
Homework6 Valentine Shimansky

### PR DESCRIPTION
**Pull request by MrRaisRay**

### CODE REVIEW

File: HW6/Runner.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from HW6.Num_n_Square import char_count
+from HW6.Num_n_Square import num_n_square
+from HW6.MoreCountedWordFinder import more_counted_word
+
+
+def runner(inner_lst):
```
Comment (belovmd) [inline HW6/Runner.py:6]: there is more generic way using getattr. in your approach we can work only with this particular functions

File: HW6/RolledLst_DataLog.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+
+
+def loger(func):
+    def wrapper(dec_lst):
+        print(func(dec_lst))
+        log = open('RolledLst.log', 'r').read()
```
Comment (belovmd) [inline HW6/RolledLst_DataLog.py:8]: you forgot to close the file

File: HW6/RolledLst_DataLog.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        log += func(dec_lst) + '\n'
+        open('RolledLst.log', 'w').write(log)
```
Comment (belovmd) [inline HW6/RolledLst_DataLog.py:10]: we can just append file

File: HW6/IMDB.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+imdb = open('./ratings.list', 'r')
+file_str = imdb.read()
+start_point = file_str.find('      00000')
+end_point = file_str.find('\n\nBOTTOM 10')
+top_250 = file_str[start_point:end_point:]
+lst_top_250 = top_250.split('\n')
```
Comment (belovmd) [inline HW6/IMDB.py:7]: it's not clear how many lines would be in this file.

File: HW6/IMDB.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+for el in lst_top_250:
+    el = el.replace('  ', ' ')
+    el = el.replace('  ', ' ')
+    inner_start_point = el.find('.') + 3
+    inner_end_point = el.find('(')
+    top250_str += el[inner_start_point:inner_end_point - 1:] + '\n'
+    str_info = el.replace(el[inner_start_point:inner_end_point - 1:], '')
+    lst_info = str_info.split()
+    lst_info[3] = lst_info[3].replace('(', '').replace(')', '')
+    if dct_ratings.get(lst_info[2]) is None:
```
Comment (belovmd) [inline HW6/IMDB.py:23]: what are these 2, 3 magic numbers?

File: HW6/IMDB.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+open('./ratings.txt', 'w').write(str(str_ratings))
+for year in range(1900, 2019,):
+    str_years += str(year)
+    str_years += '+'
+    for key in dct_years.keys():
+        if str(year) == key:
+            str_years += '+' * (dct_years[key] + 1)
+            break
+    str_years += '\n'
+open('./years.txt', 'w').write(str(str_years))
```
Comment (belovmd) [inline HW6/IMDB.py:52]: please don't forget to close these files

**Review by belovmd — CHANGES_REQUESTED** [review]
1. please fix code style
2. please remove this huge file
3.please add result files for IMDB task

File: HW6/IMDB.py (hunk 5 above)
Comment (MrRaisRay) [reply HW6/IMDB.py:23]: It's position of year and rank date in list

**Review by belovmd — COMMENTED** [review]
missed years.txt files. 
previous review issues. 
failed codestyle
6

**Review by belovmd — COMMENTED** [review]
previous review issues.
failed codestyle
6.5

**Pull request closed without merging**